### PR TITLE
Fixes the express supply console so ghosts can't order shit anymore

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Title

## Why It's Good For The Game
Fixes a bug

## Changelog
:cl:
fix: Fixes the express supply console so ghosts can no longer order stuff/make beacons.
/:cl:

### CODE REVIEW

**Comment by bloons3** [comment]
Should close https://github.com/BeeStation/BeeStation-Hornet/issues/1337

File: code/modules/cargo/expressconsole.dm [hunk 1]
```diff
@@ -156,21 +156,21 @@
 			var/name = "*None Provided*"
 			var/rank = "*None Provided*"
 			var/ckey = usr.ckey
-			if(ishuman(usr))
+			if(ishuman(usr) && isliving(usr))
```
Comment (ike709) [inline code/modules/cargo/expressconsole.dm:159]: Unnecessary, remove.

File: code/modules/cargo/expressconsole.dm [hunk 2]
```diff
@@ -156,21 +156,21 @@
 			var/name = "*None Provided*"
 			var/rank = "*None Provided*"
 			var/ckey = usr.ckey
-			if(ishuman(usr))
+			if(ishuman(usr) && isliving(usr))
 				var/mob/living/carbon/human/H = usr
 				name = H.get_authentification_name()
 				rank = H.get_assignment(hand_first = TRUE)
-			else if(issilicon(usr))
+			else if(issilicon(usr) && isliving(usr))
```
Comment (ike709) [inline code/modules/cargo/expressconsole.dm:163]: Unnecessary, remove.

**Review by ike709 — CHANGES_REQUESTED** [review]
This really isn't the correct way to go about fixing this.

**Comment by PowerfulBacon** [comment]
Can there not be 1 is living check at the start?

**Comment by bloons3** [comment]
Wouldn't that mess with admin/silicon interaction?

**Pull request closed without merging**